src/Microsoft.AspNetCore.SignalR.Specification.Tests/RedisHubLifetimeManagerTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Internal;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.AspNetCore.SignalR.Redis;
+using Microsoft.AspNetCore.SignalR.Tests;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Specification.Tests
+{
+    public class RedisHubLifetimeManagerTests : ScaleoutHubLifetimeManagerTests
+    {
+        private TestRedisServer _server;
+
+        public override HubLifetimeManager<MyHub> CreateNewHubLifetimeManager()
+        {
+            _server =  new TestRedisServer();
+
+            return CreateLifetimeManager(_server);
+        }
+
+        public override HubLifetimeManager<MyHub> CreateNewHubLifetimeManagerFromExistingServer()
```

Comment:
Seems odd, why not just use the `CreateNewHubLifetimeManager` function and let the user decide if they want to cache an internal server or not